abel/tools/analytical.py

```diff
@@ -305,3 +306,52 @@ def sym_abel_step_1d(r, A0, r0, r1):
         d[:, mask] = abel_step_analytical(np.abs(r[mask]), A0, r0, r1)
 
     return d
+
+
+def scaling(angle, amp=0.1, phase=2):
+    # define a simple scaling that will squish a circle into a "flower"
+    return 1 + amp*(np.sin(phase*angle)**4)
+
+
+def flower_distort(IM, amp=0.1, phase=2):
```

Comment:
@DanHickstein that is pretty profound!

Not quite `1 + amp*np.sin(freq*theta)**4` ;-)

Removed `def flower_distort` from `abel/tools/analytical.py` and added your function suggestion to `examples/example_circularize.py`, `tests/test_circularize.py`.